content/pt-br/docs/concepts/storage/volume-health-monitoring.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Monitoramento de Saúde do Volume
+content_type: concept
+weight: 100
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+{{< glossary_tooltip text="CSI" term_id="csi" >}} O monitoramento de saúde de volume CSI permite que os drivers CSI detectem condições anormais de volume dos sistemas de armazenamento subjacentes e os relatem como eventos em {{< glossary_tooltip text="PVCs" term_id="persistent-volume-claim" >}} ou {{< glossary_tooltip text="Pods" term_id="pod" >}}.
```

Comment:
This sentence sounds good, except that this "CSI" is out of place at the beginning of this sentence. Let's remove it and move the tooltip to the first occurrence of CSI.

Reply:
I didn't pay attention to the position of the CSI, corrected